server/eclipse-project/src/main/java/us/freeandfair/corla/csv/DominionCVRExportParser.java

```diff
@@ -516,8 +664,28 @@ private CastVoteRecord extractCVR(final CSVRecord the_line) {
         index = index + 1;
       }
       // if this contest was on the ballot, add it to the votes
+      try {
       if (present) {
-        contest_info.add(new CVRContestInfo(co, null, null, votes));
+        if(isIRV) {
+            // If it is IRV, convert it into an ordered list of names (without parentheses), then
+            // store.
```

Comment:
does getValidIntentAsOrderedList change an IRVChoices instance? if not, can make irvVotes final